docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -490,6 +490,14 @@ spec:
 > default interface type to `masquerade`, and disabling the `bridge`
 > type for pod network, as shown in the example above.
 
+> **Note:** Migrating a VM with secondary interfaces that uses bridge 
+> binding may cause long periods of traffic disruption. This occurs when
+> the interface is defined with an explicit MAC address (manually or 
+> automatically through KubeMacPool) on nodes that have IPv6 enabled.
```

Comment:
I wouldn't mention the low level details about the MAC and the IPv6. 
I would just say - "One of the reasons is that the target pod interface announces about its existence before the migration is completed."

Reply:
Setting custom MAC and having IPv6 enabled is the terms for this issue in the first place.
I can add what you suggest,  but I think its worth mentioning the terms, WDYT?

Reply:
I think that those are low lover implementation details that may confuse the reader.

Reply:
Done